fiddler/manifest.json

```
@@ -0,0 +1,56 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ee617671-508e-4bb3-ba25-8815b11a16aa",
+  "app_id": "fiddler",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+    "Category::Alerting",
+    "Category::Monitoring",
+    "Category::Metrics",
+    "Offering::Integration"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Fiddler integration to bring model metrics into DataDog",
```

```suggestion
    "description": "Fiddler integration to bring model metrics into DataDog",
```
This description gets displayed on the tile preview - we recommend changing it to something more specific about what Fiddler is and the type of data the integration sends in

---

Done.